composer.json

```diff
@@ -31,5 +30,10 @@
 		"psr-4": {
 			"MASK\\Mask\\": "Classes/"
 		}
+	},
+	"extra": {
+		"branch-alias": {
+			"dev-master": "3.1.1dev"
```

Comment:
Just for reference: you wouldn't use a concrete version as branch alias but 3.1-dev or 3.x-dev

Now you need to ensure to raise the branch alias if a new version was released!